pkg/jvscrypto/key_hosting.go

```diff
@@ -19,16 +19,19 @@ import (
 	"fmt"
 	"net/http"
 
+	"cloud.google.com/go/firestore"
 	kms "cloud.google.com/go/kms/apiv1"
 
 	"github.com/abcxyz/jvs/pkg/config"
+	fsutil "github.com/abcxyz/jvs/pkg/firestore"
 	"github.com/abcxyz/pkg/cache"
 	"github.com/abcxyz/pkg/logging"
 )
 
 // KeyServer provides all valid and active public keys in a JWKS format.
 type KeyServer struct {
 	KMSClient       *kms.KeyManagementClient
+	FsClient        *firestore.Client
```

Comment:
This should be `FirestoreClient`

Reply:
+1. `FsClient` makes me think of filesystem related stuff